### DISCUSSION TITLE
Why this window title font scales with the terminal font?

### DISCUSSION THREAD

**Discussion by shivangrathore** [kovidgoyal/kitty#7677] [General]
Is there any way to opt out from this behaviour?
Also I would like this text in the middle.
![image](https://github.com/user-attachments/assets/7e27544a-2ee6-419b-9f73-6e3378f8e8df)

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/commit/62bd6c88e99a82bedeece3c8bca15dab4b629098

**Reply by shivangrathore** [reply]
Thank you :beers:
